pontoon/sync/tasks.py

```diff
@@ -69,7 +69,7 @@ def sync_project(self, project_pk, sync_log_pk, no_pull=False, no_commit=False,
 
         return
 
-    obsolete_vcs = perform_sync_project(db_project, now)
+    obsolete_vcs, obsolete_resources = perform_sync_project(db_project, now)
```

Comment:
`obsolete_vcs_entities, obsolete_vcs_resources` (and also in the rest of the file).